ios-trivia-game/ViewControllers/GameOptionsViewController.swift

```diff
@@ -33,3 +77,34 @@ class GameOptionsViewController: UIViewController {
     */
 
 }
+
+extension GameOptionsViewController: UIPickerViewDelegate, UIPickerViewDataSource {
+    
+    // The number of columns of data
+    func numberOfComponents(in pickerView: UIPickerView) -> Int {
+        return 1
+    }
+    
+    // The number of rows of data
+    func pickerView(_ pickerView: UIPickerView, numberOfRowsInComponent component: Int) -> Int {
+        if pickerView.tag == 1 {
```

Comment:
This seems like code-smell to me. Especially when we are using the tag in both the pickerView methods. Can we abstract out what tag 1 and tag 2 are? Perhaps make them into a constant. Otherwise, it'd be a pain in the future to maintain.

It's not a big concern, but it'd be nice to have if we can fix it.